### ISSUE TITLE
✨ Support pushing program-defined surveys to a filtered set of users

### ISSUE THREAD

**Issue by shankari** [e-mission/e-mission-docs#976]
We used to have the ability to push a program-defined survey to users on demand.

The canonical use case that motivated it was:
- city wants to put a new bike lane along a particular stretch of road
- it has a large active base on OpenPATH users
- it pushes out a survey to the people who frequently travel along that road asking them to weigh on in which class of bike lane to use

Because this is linked with their OpenPATH data, we don't have to waste time asking them how often they bike along that road, what time of day, for what purpose, do they ever drive along, etc - we can get all that directly from their OpenPATH data. The survey can focus on what various levels of bicycle infrastructure will make them feel.

In other words, let the phone do what it does best (longitudinal, relentless recording of data) and let people do what they do best (represent their emotions and perceptions).

This code has bitrotted as part of several changes, we should resurrect it, and make it self-service through the admin dashboard.
